### ISSUE TITLE
Skewed dataset?

### ISSUE THREAD

**Issue by j16sdiz** [apavlinovic/go-hub-spawn-db-feb-2017#1]
Refer to the article here: https://pokemongohub.net/generation-2/analysis-time-day-influences-different-pokemon-type-spawns/

You had 96728 spawn at hour 0, but only 76705 spawn at hour 3. Does it not make sense? We all knew the spawn points are fixed and they spawns every hour.
If they came from different spawnpoints, the data should be normalized before comparing.

**Comment by apavlinovic** [comment]
You are correct, the data is not normalised per spawn points. We reached out to the original provider of the data to figure out if the data set was created using bots or organic users.
